systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeCorsST.java

```diff
@@ -53,6 +76,25 @@ public class HttpBridgeCorsST extends AbstractST {
     private TestStorage suiteTestStorage;
 
     @ParallelTest
+    @TestDoc(
+        description = @Desc("This test checks if CORS handling for allowed origin works correctly in the Kafka Bridge."),
+        contact = @Contact(name = "Lukas Kral", email = "lukywill16@gmail.com"),
+        steps = {
+            @Step(value = "Set up the Kafka Bridge user and configuration", expected = "Kafka Bridge user and configuration are set up"),
+            @Step(value = "Construct the request URL and headers", expected = "URL and headers are constructed properly"),
+            @Step(value = "Send OPTIONS request to Kafka Bridge and capture the response", expected = "Response is captured from Bridge"),
+            @Step(value = "Validate the response contains expected status codes and headers", expected = "Response has correct status codes and headers for allowed origin"),
+            @Step(value = "Send GET request to Kafka Bridge and capture the response", expected = "Response is captured from Bridge for GET request"),
+            @Step(value = "Check if the GET request response is '404 Not Found'", expected = "Response for GET request is 404 Not Found")
+        },
+        useCases = {
+            @UseCase(id = "validate_cors_handling_allowed_origin")
```

Comment:
Where are these defined?

Reply:
Should we have them defined somewhere? I tried to use the use case like having the closer description of the test or suite. I assume it could help if we have a detailed description of such use cases but not sure how that helps ;) 

 ^^ @strimzi/system-test-contributors

Reply:
It seems weird to have it as `id` with snake case if it doesn't link to anything. Would just some freetext `description` be better?

Reply:
The main idea was to create MD files (as it is mentioned in the generator's repo and its README), where you will have some description of the usecase and then list of testcases (with links to particular MD files) underneath.

Reply:
Looking back to this...Would it add so much value to define all the use cases? I mean it's a lot of work where you imagine that we would end up documenting whole STs. So now I am thinking about if descriptive use cases like this are not enough for now. And if that would be a problem in future could we add those?

Reply:
Not sure if this was at me, at Lukas, or someone else. My main issue is that you introduce some use-case IDs that do not lead anywhere. So as I said, I can imagine for example:
* Removing it completely
* Changing the `id` to some `description` and make it _one sentence_ description of the use-case

It just doesn't seem make sense to have some use case ID that does not lead anywhere.

Reply:
^^ @strimzi/system-test-contributors any opinion on this?

Reply:
I guess it's useless to have use-cases there without any description what it actually covers. Because yeah, you can have one or two which are descriptive enough, but rest which will be a bit confusing when you use just the ID.
The idea of use-cases and linking between is that you will have one file with the description and the test-cases, which cover this use-case.

In case that you think it's a lot of work documenting the STs (which is the main goal of this), you should either remove it or leave it for someone else.

Also I don't think that it's good to have ID as a whole sentence. It's nice that you generate it, but you will end up creating multiple use-cases which are actually just one (and you will have to add description for all of them). So in this case, it makes more sense to go through the tests and add use-cases that are really relevant manually.

Reply:
IIRC the original idea was to have some high-level use-case with some reasonable description - I would say maybe detailed description of interaction between user and Strimzi. For example `User can migrade from Zookeeper-based Kafka cluster to KRaft based cluster` + some details, description of corner cases etc. This description would be something that will be provided as part of planning/propsal? Then the each test cases will be linked to use-cases so each use-case will be covered by N test-cases. 

The idea was also to easily find which tests covers what use-cases/features, etc.